### DISCUSSION TITLE
Como se hace las Katas

### DISCUSSION THREAD

**Discussion by lilmendez** [LaunchX-InnovaccionVirtual/CursoIntroPython#257] [Q&A] [answered]
Buenas noches vi el curso del viernes como hacer las katas, al descargar el archivo de github y ponerlo al visual code ya estaba hecho todo, entonces cual es la tarea si ya esta hecho el codigo, y otro pregunta hoy dia hay nuevas clases o ya empieza el proximo lunes, gracias.

**Comment by ArielCanedo** [comment, marked as answer]
las katas tienen dos archivos, una es la explicación y otro ejercicios.
Si bien la solución viene en los ejercicios, la idea es que tú los resuelvas
